projects/vdk-plugins/vdk-jupyter/build-job-tutorial-temp/create_tutorial.md

```diff
@@ -0,0 +1,62 @@
+# Learn How to Create  a Data Job through the Jupyter UI
+Time Commitment: About 5-10 minutes.
+
+## Prerequisites
+
+Before you start with the build, please ensure you have the following:
+
+<details>
+  <summary><b>Installed Software and Services</b></summary>
```

Comment:
Installed Software and Services seem generic and scary like it's a lot of complex stuff

What they really need is some Python VDK distribution installed (like quickstart-vdk) + the jupyter plugins. 

So I would name the section "Some Python VDK distribution installed"

Reply:
I will keep it this way since it is like that in other tutorials